DJHelper/DJHelper/View Controllers/EventPlaylistViewController.swift

```diff
@@ -403,4 +408,62 @@ extension EventPlaylistViewController: UITableViewDataSource {
         }
         return cell
     }
+
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if isGuest == false {
+            if editingStyle == .delete {
+
+                switch currentSongState {
+                case .requested:
+                    let song = requestedSongs[indexPath.row]
+                    deleteTrackFromRequestList(song: song)
```

Comment:
I like how these methods are very clearly named.  "deleteTrackFromRequestList" is super obvious.